src/components/common/SelectInput.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from 'react';
+import { arrayOf, bool, func, string } from 'prop-types';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { parseInputErrors } from 'utils/helpers';
+import '../../styles/_variables.scss';
+
+const SelectInput = ({
+  label,
+  name,
+  value,
+  onChange,
+  errors,
+  active,
+  touched,
+  options,
+  ...props
+}) => {
+  // Register field in the form
+  useEffect(() => {
+    onChange({ target: { value } }, true);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const intl = useIntl();
+
+  return (
+    <div>
+      <div className="input-container">
+        {label && (
+          <label className="input-label" htmlFor={name}>
+            {label}
+          </label>
+        )}
+        <select name={name} value={value} onChange={onChange} {...props}>
+          {options.map(opt => {
+            return (
+              <option key={opt.id} value={opt.value}>
```

Comment:
Maybe the `option` can be a new component too? And we can move this to a `SelectInput` folder and rename this file as `index.js`. So the `Option` component can be inside this new folder

Reply:
Cool, I've extracted each option of the SelectInput to a new Component and moved to this new folder structure. The thing is idk to what extent is it worth it, because even though you have a much more compact component, code in SelectInput still remains pretty similar, and now you have one more component to take care of. Would you check the new structure and let me know what you think?

Reply:
IMO is better to have different components with less logic, so they can be tested easily and probably reused in many places.
I think we shouldn't worry about "take care" of this components, tests can help with this.